sqlconnect/internal/redshift/config.go

```diff
@@ -30,6 +30,10 @@ type Config struct {
 	SecretAccessKey string `json:"secretAccessKey"`
 	SessionToken    string `json:"sessionToken"`
 
+	RoleARN               string        `json:"roleARN"`
+	ExternalID            string        `json:"externalID"`
+	RoleARNExpiryDuration time.Duration `json:"roleARNExpiryDuration"` // default: 1h
```

Comment:
[By default the client's expiry is set to 15 minutes](https://github.com/aws/aws-sdk-go-v2/blob/main/credentials/stscreds/assume_role_provider.go#L139-L146). 
Do we have a reason for overriding this default behaviour?

Reply:
We have seen single queries taking around more than 15 mins for customers. (Max around 25-30 mins).

Reply:
https://github.com/aws/aws-sdk-go-v2/blob/main/aws/credential_cache.go#L98

Reply:
for reference, profiles queries can take more than 2 hours :)

Reply:
Will remove it then.

Reply:
We may keep the config option, just don't override the default value.